### ISSUE TITLE
RPortd built with golang 1.20+ leads to different generated fingerprint

### ISSUE THREAD

**Issue by ben-alphabeans** [realvnc-labs/rport#1037]
I have updated rportd to 0.9.12 and found the fingerprint changed, and my clients cannot connect to it.
I have tried to build it myself and found different behavior between go 1.19 and 1.20.
`ecdsa.GenerateKey()` returns differently between both golang version.

I don't know if this is a bug or just some changes that must happen. 
Thanks.

**Comment by thorstenkramm** [comment]
@ben-alphabeans Yes. This is, in fact, an issue, and we don't have a long-term solution yet.
Two immediate solutions:
1. Compile with go 1.19. Rport 0.9.12 doesn't require go 1.20
2. Use [our 0.9.12 binaries](https://github.com/realvnc-labs/rport/releases/tag/0.9.12) that have been compiled with go 1.19 and fingerprints won't change.
